gobblin-core/src/main/java/gobblin/writer/http/LimiterHttpClient.java

```diff
@@ -0,0 +1,44 @@
+package gobblin.writer.http;
+
+import java.io.IOException;
+import com.codahale.metrics.Meter;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import gobblin.http.HttpClient;
+import gobblin.util.limiter.Limiter;
+
+@Slf4j
+@AllArgsConstructor
+public class LimiterHttpClient<RQ, RP> implements HttpClient<RQ, RP> {
```

Comment:
@ibuenros Because the limiter has handled NOOP operation, I don't think we need additional wrapper. Removed for now.
@zxcware Throttling can be applied at different layer. At this point, if retry-sending needs to be throttled, it should be in sendRequest because it is the real TX operation.